src/test/kotlin/uk/gov/justice/digital/hmpps/courtlistsplitter/service/TelemetryServiceTest.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.courtlistsplitter.service
+
+import com.microsoft.applicationinsights.TelemetryClient
+import com.nhaarman.mockitokotlin2.verify
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+
+@ExtendWith(MockitoExtension::class)
+internal class TelemetryServiceTest {
+
+  @Mock
+  private lateinit var telemetryClient: TelemetryClient
+
+  @InjectMocks
+  private lateinit var telemetryService: TelemetryService
+
+  @Test
+  fun `should track event`() {
+
+    telemetryService.trackEvent(TelemetryEventType.COURT_LIST_RECEIVED)
+
+    verify(telemetryClient).trackEvent("PiCCourtListReceived")
```

Comment:
I assume the customFields will follow as part of [PIC-1335](https://dsdmoj.atlassian.net/browse/PIC-1335)?

Reply:
I think so, yes. I think we may need to revisit the events though with the new architecture